dali/operators/image/color/brightness_contrast.h

```diff
@@ -95,15 +90,21 @@ class BrightnessContrastOp : public Operator<Backend> {
   }
 
   void AcquireArguments(const workspace_t<Backend> &ws) {
-    this->GetPerSampleArgument(brightness_, "brightness", ws);
-    this->GetPerSampleArgument(brightness_shift_, "brightness_shift", ws);
-    this->GetPerSampleArgument(contrast_, "contrast", ws);
+    auto curr_batch_size = ws.GetInputBatchSize(0);
+    this->GetPerSampleArgument(brightness_, "brightness", ws, curr_batch_size);
+    this->GetPerSampleArgument(brightness_shift_, "brightness_shift", ws, curr_batch_size);
+    this->GetPerSampleArgument(contrast_, "contrast", ws, curr_batch_size);
 
     input_type_ = ws.template InputRef<Backend>(0).type().id();
-    output_type_ =
-        output_type_arg_ != DALI_NO_TYPE
-        ? output_type_arg_
-        : input_type_;
+    output_type_ = output_type_arg_ != DALI_NO_TYPE ? output_type_arg_ : input_type_;
+  }
+
+  void KMagResize(int num_threads, int batch_size) {
```

Comment:
```suggestion
  void KMgrResize(int num_threads, int batch_size) {
```
`Mag` is not an abbreviation of "manager".